### ISSUE TITLE
Update for WCAG Technique F87 (Obsolete)

### ISSUE THREAD

**Issue by kentarou-fukuda** [eclipse-actf/org.eclipse.actf#13]
In WCAG 2.2, F87 becomes obsolete as below.

https://www.w3.org/WAI/WCAG22/Techniques/failures/F87

We need to update our checkitem's categolization and description for F87.
